vhd/packer/install-dependencies.sh

```diff
@@ -422,6 +423,8 @@ pullContainerImage "docker" "busybox"
 echo "  - busybox" >> ${VHD_LOGS_FILEPATH}
 
 K8S_VERSIONS="
+1.18.2
+1.18.1
```

Comment:
these needs to go to AKS VHD 
https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/packer/install-dependencies.sh